src/components/SidenavBase/SidenavBase.style.ts

```diff
@@ -27,40 +22,29 @@ type SidebarNavLinkProps = {
 } & ExpandableElementProps &
   LinkProps
 
-export const SidebarNav = styled.nav<SidebarNavProps>`
+export const SidebarNav = styled.nav<ExpandableElementProps>`
   position: fixed;
   top: 0;
   bottom: 0;
   height: 100%;
   z-index: ${zIndex.sideNav};
-  width: ${({ expanded }) => (expanded ? `${EXPANDED_SIDENAVBAR_WIDTH}px` : 'var(--sidenav-collapsed-width)')};
+  width: ${({ expanded }) => (expanded ? `${EXPANDED_SIDENAVBAR_WIDTH}px` : 'var(--size-sidenav-width-collapsed)')};
   transition: width ${transitions.timings.regular} ${transitions.easing};
   display: grid;
   grid-template-rows: auto auto minmax(0, 1fr) auto;
   overflow: hidden;
-  color: ${colors.white};
-  background-color: ${({ isStudio }) => (isStudio ? colors.gray[800] : colors.gray[700])};
+  color: ${colors.gray[50]};
+  background-color: ${colors.gray[800]};
 `
 
 export const LogoLink = styled(Link)`
   display: flex;
-  margin-top: 24px;
+  align-items: center;
+  height: var(--size-topbar-height);
   margin-left: 80px;
   text-decoration: none;
 `
 
-const logoStyles = css`
-  height: ${sizes(8)};
-`
-
-export const JoystreamLogo = styled(SvgJoystreamFullLogo)`
-  ${logoStyles}
-`
-
-export const StudioLogo = styled(SvgStudioFullLogo)`
-  ${logoStyles}
-`
-
 export const SidebarNavList = styled.ul`
   margin-top: 56px;
```

Comment:
According to the designs, this should be `28px

Reply:
Yes, updated now. Also did some tweaks to spacing and alignment. Let me know if you see anything else